tools_galaxyp.yaml

```diff
@@ -1,158 +1,925 @@
----
 install_repository_dependencies: true
 install_resolver_dependencies: true
 install_tool_dependencies: false
 
 tools:
-  - name: uniprotxml_downloader
+  - name: retrieve_ensembl_bed
     owner: galaxyp
     tool_panel_section_label: Get Data
 
-  - name: dbbuilder
+  - name: ms_wiff_loader
     owner: galaxyp
     tool_panel_section_label: Get Data
 
-  - name: unipept
+  - name: regex_find_replace
     owner: galaxyp
-    tool_panel_section_label: Get Data
+    tool_panel_section_label: Text Manipulation
+
+  - name: concatenate_multiple_datasets
+    owner: mvdbeek
+    tool_panel_section_label: Text Manipulation
+
+  - name: filter_by_substring_match
+    owner: alan-blakely
+    tool_panel_section_label: Filter and Sort
 
-  - name: map_peptides_to_bed
+  - name: column_join
+    owner: jjohnson
+    tool_panel_section_label: Join, Subtract and Group
+
+  - name: gffcompare_to_bed
     owner: galaxyp
     tool_panel_section_label: Convert Formats
 
-  - name: idconvert
+  - name: bed_to_protein_map
     owner: galaxyp
     tool_panel_section_label: Convert Formats
 
-  - name: idpquery
+  - name: translate_bed_sequences
     owner: galaxyp
     tool_panel_section_label: Convert Formats
 
-  - name: thermo_raw_file_converter
+  - name: peca
+    owner: caleb-easterly
+    tool_panel_section_label: Statistics
+
+  - name: limma_norm
+    owner: caleb-easterly
+    tool_panel_section_label: Statistics
+
+  - name: concat_fasta_files
+    owner: mandorodriguez
+    tool_panel_section_label: FASTA manipulation
+
+  - name: mz_to_sqlite
     owner: galaxyp
-    tool_panel_section_label: Convert Formats
+    tool_panel_section_label: MS Data Conversion
 
-  - name: msconvert_nix
+  - name: mgf_formatter
     owner: galaxyp
-    tool_panel_section_label: Convert Formats
+    tool_panel_section_label: MS Data Conversion
 
-  - name: msconvert
+  - name: msconvert_win
     owner: galaxyp
-    tool_panel_section_label: Convert Formats
+    tool_panel_section_label: MS Data Conversion
 
-  - name: fasta_merge_files_and_filter_unique_sequences
+  - name: ms_data_converter
     owner: galaxyp
-    tool_panel_section_label: FASTA/FASTQ manipulation
+    tool_panel_section_label: MS Data Conversion
 
-  - name: peptideshaker
+  - name: proteinpilot
     owner: galaxyp
-    tool_panel_section_label: Proteomics
+    tool_panel_section_label: MS Data Conversion
 
-  - name: lfq_protein_quant
+  - name: validate_fasta_database
     owner: galaxyp
-    tool_panel_section_label: Proteomics
+    tool_panel_section_label: Sequence Database Tools
+
+  - name: fastq_join
```

Comment:
This tool is also not needed imho, it can be replaced by https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/flash/flash/1.2.11.3 I think.